docs/rucio.md

```diff
@@ -177,7 +177,7 @@ The command should show the `rucio-client` container running.
 Have your certificate ready and divided into two files named ~/.globus/userkey.pem and ~/.globus/usercert.pem. 
 
 ```bash
-docker run --user root -e RUCIO_CFG_CLIENT_X509_PROXY=/tmp/x509up -e RUCIO_CFG_AUTH_TYPE=x509_proxy -e RUCIO_CFG_ACCOUNT=<myrucioname> -v ~/.globus/usercert.pem:/opt/rucio/etc/client.crt -v ~/.globus/userkey.pem:/opt/rucio/etc/client.key -v ./<path_to_local_data_directory>:/home/<path_to_local_data_directory> -it --name=rucio-client ghcr.io/vre-hub/vre-rucio-client
+docker run --user root -e RUCIO_CFG_CLIENT_CLIENT_X509_PROXY=/tmp/x509up -e RUCIO_CFG_CLIENT_AUTH_TYPE=x509_proxy -e RUCIO_CFG_CLIENT_ACCOUNT=<myrucioname> -v ~/.globus/usercert.pem:/opt/rucio/etc/client.crt -v ~/.globus/userkey.pem:/opt/rucio/etc/client.key -v ./<path_to_local_data_directory>:/home/<path_to_local_data_directory> -it --name=rucio-client ghcr.io/vre-hub/vre-rucio-client
```

Comment:
I think you have doubled `CLIENT`  in the `RUCIO_CFG_CLIENT_CLIENT_X509_PROXY` arg

Reply:
No no, it is ok. The first occurence of `CLIENT` is because the option corresponds to the client section of the config file and the second occurrence is because the option's name itself is `CLIENT_X509_PROXY`.

Reply:
The syntax to use is `RUCIO_CFG_<cfg-section>_<cfg-option>`.